inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tile.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+typedef std::vector<size_t> tileSpecificParams;
```

Comment:
Minor comment, Type name should look as `TileSpecificParams`

Reply:
thanks, changed